macros/pivot_json_extract.sql

```diff
@@ -1,9 +1,13 @@
 {% macro pivot_json_extract(string, list_of_properties) %}
 
 {%- for property in list_of_properties -%}
+{%- if property is not string -%}
```

Comment:
I think we discussed during that quick show and tell that it may be better to use the following. This way you must be using a dictionary instead of not string. This way if customers need to pass an int or something, they aren't hit with an error.
```suggestion
{%- if property is mapping -%}
```

Reply:
Oh yes that's right-- testing it now